definitions/tools/manta_somatic.cwl

```diff
@@ -2,13 +2,17 @@
 
 cwlVersion: v1.0
 class: CommandLineTool
-label: "Set up and execute manta using helper script"
+label: "Set up and execute manta"
 
 requirements:
     - class: DockerRequirement
       dockerPull: mgibio/manta_somatic-cwl:1.5.0
     - class: InlineJavascriptRequirement
     - class: ShellCommandRequirement
+    - class: ResourceRequirement
+      coresMin: 2
```

Comment:
Should we use a few more cores? In the GMS we were running with 12 and 24GB of RAM. I have not performed a benchmark but I presume there was a reason to use a large number of threads and RAM in the GMS.